code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -6,6 +6,7 @@
 	item_state = "eng_helm"
 	max_integrity = 300
 	armor = list("melee" = 10, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 10, "bio" = 100, "rad" = 75, "fire" = 50, "acid" = 75)
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
```

Comment:
hardsuits are a subclass of spacesuits so they already have these flags

Reply:
This is the helmet. It doesn't have these flags.

Reply:
/obj/item/clothing/head/helmet/space in _spacesuits.dm, line 21

Reply:
ah true. Aight yeah I guess it didn't matter.